internal/tests/cli/boundary/credentials.bats

```diff
@@ -91,9 +91,9 @@ export NEW_KV_CREDENTIAL='test-kv'
   [ "$status" -eq 1 ]
 }
 
-@test "boundary/credential-stores: can not read deleted $NEW_KV_CREDENTIAL credential" {
+@test "boundary/credential-stores: can not read deleted $NEW_JSON_CREDENTIAL credential" {
   local csid=$(credential_store_id $NEW_STORE $DEFAULT_P_ID)
-  local cid=$(credential_id $NEW_KV_CREDENTIAL $csid)
+  local cid=$(credential_id $NEW_JSON_CREDENTIAL $csid)
   run read_credential $cid
   echo "$output"
   [ "$status" -eq 1 ]
```

Comment:
Will we add a test for setting a `-object` and `-kv` flag at the same time separately?